src/app/users/getServiceUsers.js

```diff
@@ -1,24 +1,96 @@
-const { extractPageParam, extractPageSizeParam } = require('../utils');
+const { extractPageParam, extractPageSizeParam, extractStatusParam, extractFromParam, extractToParam } = require('../utils');
 const { listServiceUsers } = require('../../infrastructure/organisations');
 const { usersByIds } = require('../../infrastructure/directories');
+const { directories } = require('login.dfe.dao');
 
 const listUsers = async (req, res) => {
   let page;
   let pageSize;
+  let status;
+  let to;
+  let from;
+  let fromDate;
+  let toDate;
+
   try {
     page = extractPageParam(req);
     pageSize = extractPageSizeParam(req);
+    status = extractStatusParam(req);
+    to = extractToParam(req);
+    from = extractFromParam(req);
+
+    // for future implementation
+    // if (status && status !== '1' && status !== '0') {
```

Comment:
instead of making it 0 in your code, use this commented out block, but return error if value passed in is not 0